src/mscorlib/shared/System/DataMisalignedException.cs

```diff
@@ -32,5 +34,9 @@ public DataMisalignedException(String message, Exception innerException)
         {
             HResult = HResults.COR_E_DATAMISALIGNED;
         }
+
+        internal DataMisalignedException(SerializationInfo info, StreamingContext context) : base(info, context)
```

Comment:
Is there any value in these empty implementations? The base call would happen anyway? I think it's fine though since desktop does it.

Reply:
The ctor is necessary to instantiate the type :)

Reply:
Doh, I misread as GetObjectData.